src/js/media-chrome-range.js

```diff
@@ -182,10 +183,12 @@ template.innerHTML = `
       background-color: #777;
     }
   </style>
-  <div id="background"></div>
-  <div id="pointer"></div>
-  <div id="hoverzone"></div>
-  <input id="range" type="range" min="0" max="1000" step="any" value="0">
+  <div id="container">
```

Comment:
added a container div here which makes it easier to take in account the padding that is added on media-time-range
#background, #pointer, etc which are absolutely positioned don't have to take in to account that padding anymore